src/Raven.Studio/typescript/test/stubs/DatabasesStubs.ts

```diff
@@ -951,4 +951,24 @@ return docs[0];`,
             },
         ];
     }
+
+    static getIdentities(length = 2): Record<string, number> {
+        const object: Record<string, number> = {};
+        for (let i = 1; i < length; i++) {
```

Comment:
how many identities do I get when I call it with length = 5?

Reply:
due to line: 
`for (let i = 1; i < length; i++) {`

**<** means that we will have object with **4 keys**, which may be a little unclear, and I would add an **equal sign** here so that **the length equals the number of keys in the object**